doc/api/crypto.md

```diff
@@ -2490,14 +2493,14 @@ changes:
 
 Checks whether the certificate matches the given email address.
 
+If the `'subject'` option is undefined or set to `'default`', the certificate
```

Comment:
```suggestion
If the `'subject'` option is undefined or set to `'default'`, the certificate
```

Reply:
Argh, thanks, I missed this one in #41569. Unfortunately, this PR is semver-major, so fixing it here would prevent it from being backported. https://github.com/nodejs/node/pull/41613.